frontend/src/v5/ui/controls/resizableTableContext/resizableTableContext.tsx

```diff
@@ -58,69 +66,77 @@ ResizableTableContext.displayName = 'ResizeableColumns';
 interface Props {
 	children: any;
 	columns: TableColumn[];
-	hiddenColumns: string[];
 	columnGap?: number;
 }
-export const ResizableTableContextComponent = ({ children, columns: inputColumns, hiddenColumns: inputHiddenColumns, columnGap = 0 }: Props) => {
-	const [columns, setColumns] = useState([...inputColumns]);
-	const [hiddenColumns, setHiddenColumns] = useState(inputHiddenColumns);
+export const ResizableTableContextComponent = ({ children, columns, columnGap = 0 }: Props) => {
+	const [columnsWidths, setColumnsWidths] = useState<Record<string, number>>({});
+	const [hiddenColumns, setHiddenColumns] = useState([]);
```

Comment:
Do you see any obvious downside to having `visible` (or `hidden`) in  TableColumn?